tracer/src/Datadog.Tracer.Native/debugger_constants.h

```diff
@@ -36,6 +36,48 @@ const WSTRING skip_assemblies[]{WStr("mscorlib"),
                                 WStr("ISymWrapper"),
                                 WStr("Datadog.Trace"),
                                 WStr("MSBuild")};
+
+const WSTRING general_error_message = WStr("Failed to instrument the method.");
+
+inline WSTRING GetGenericErrorMessageWithErrorCode(short errorCode)
+{
+    return general_error_message + WStr(" [Error Code: ") + shared::ToWSTRING(errorCode) + WStr("]");
+}
+
+const WSTRING invalid_probe_method_already_instrumented =
+    WStr("Dynamic Instrumentation failed to install the probe because the corresponding method is already instrumented by another product.");
+const WSTRING invalid_method_probe_probe_is_not_supported =
+    WStr("The method where the probe should have been placed is not supported for now.");
+const WSTRING line_probe_il_offset_lookup_failure =
+    WStr("There was a failure in determining the exact location where the line probe was supposed to be placed.");
+const WSTRING line_probe_in_async_generic_method_in_optimized_code =
+    WStr("Placing line probes in async generic methods in Release builds is currently not supported. Please note that this limitation only applies to optimized code.");
```

Comment:
I'm not sure this information will be helpful to the customer. Maybe shorten it somehow? Or use an error code like in other cases?
Do you think that is understandable?

Reply:
I tend to agree with you, I'll get rid of the last part "Please note that this limitation only applies to optimized code".

Reply:
Shortened in 7bdbd73b1bd67c0a2e90ac3c6adb3d7300cd819d.